spec/support/vcr_cassettes/rrd/lighthouse_observation.yml

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
Is there a reason this was created in the rrd cassettes and not ours? Back when Alastair was on the team, we've had a "rule" that we always copy over vcrs from other modules, but it looks like you are intentionally creating it elsewhere for our use. I'm open to changing that rule because it feels unnecessary to me, but I'm curious if this was an intentional decision.

Reply:
I added it here because the cassette is for the client which is not owned by us. The other cassette's in this directory are for the other functions in the client so I thought it was the natural place to put this

Reply:
I'm good with that. I've always been conflicted about this "rule".